platform/admin/core/admin-core-insecuredefaults/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -9,18 +10,26 @@
  * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
- **/ -->
-<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+ **/
+ -->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
 
-    <reference id="eventAdmin" interface="org.osgi.service.event.EventAdmin"/>
-    <bean id="insecureDefaultsCollector"
-          class="org.codice.ddf.admin.insecure.defaults.service.InsecureDefaultsCollector">
-        <argument ref="eventAdmin"/>
-    </bean>
-    <service ref="insecureDefaultsCollector" interface="java.lang.Runnable">
-        <service-properties>
-            <entry key="decanter.collector.name" value="insecureDefaults"/>
-        </service-properties>
-    </service>
+<reference id="eventAdmin" interface="org.osgi.service.event.EventAdmin"/>
+<bean id="insecureDefaultsCollector"
+  class="org.codice.ddf.admin.insecure.defaults.service.InsecureDefaultsCollector">
+    <argument ref="eventAdmin"/>
+</bean>
+<service ref="insecureDefaultsCollector" interface="java.lang.Runnable">
+    <service-properties>
+        <entry key="decanter.collector.name" value="insecureDefaults"/>
+    </service-properties>
+</service>
 
+<bean id="usersPropertiesDeletionFlag"
+  class="org.codice.ddf.admin.insecure.defaults.service.UsersPropertiesFileValidator">
+    <cm:managed-properties
+      persistent-id="org.codice.ddf.admin.insecure.defaults.service.UsersPropertiesFileValidator"
+      update-strategy="container-managed"/>
+</bean>
```

Comment:
❓ Where does this bean get wired into the `insecureDefaultsCollector` service?

Reply:
It's not. Should it be?